ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-ec-4.13.yaml

```diff
@@ -159,6 +163,20 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: aws-ipi-ovn-hypershift-cp-p1-f4
```

Comment:
Sorry for the last minute comment, would you mind to sort the jobs ?

Reply:
@LiangquanLi930 Done, Thanks a lot

Reply:
@liangxia  Done, Thanks a lot